modules/openapi-generator/src/main/resources/JavaJaxRS/beanValidationHeaderParams.mustache

```diff
@@ -0,0 +1 @@
+{{#required}} @NotNull{{/required}}{{>beanValidationCore}} 
```

Comment:
@fujigon just a heads up that I may update this later to support `nullable` as well

Reply:
UPDATE: I've filed https://github.com/OpenAPITools/openapi-generator/pull/1679

Reply:
thanks for following up!

Reply:
@fujigon #1679 has been merged into master.